src/Razor/src/Microsoft.VisualStudio.Editor.Razor/DynamicFiles/RazorSpanMappingService.cs

```diff
@@ -80,7 +82,7 @@ internal static bool TryGetMappedSpans(TextSpan span, SourceText source, RazorCS
             if (leftOffset >= 0 && rightOffset <= 0)
             {
                 // This span mapping contains the span.
-                mappedSpan = new TextSpan(original.Start + leftOffset, (original.End + rightOffset) - (original.Start + leftOffset));
+                mappedSpan = new TextSpan(original.Start + leftOffset, original.End + rightOffset - (original.Start + leftOffset));
```

Comment:
Nit, but personally speaking I prefer the parentheses

Reply:
I'll revert this code action application. Is there an `.editorconfig` option that can be enabled to avoid this?

Reply:
I thought there were two analyzers that covered this, one for "remove unnecessary parens" and one for "remove unnecessary parens (where obvious)". I would have thought the latter wouldnt have suggested removing this, but I've no idea what our config looks like, or maybe my brain is smoother than what Roslyn considers "obvious" :)